BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -300,6 +300,40 @@ - (NSArray *)fullFileFieldsArray
     return array;
 }
 
+- (NSArray *)performantFileFieldsArray
+{
+    NSArray *array = @[BOXAPIObjectKeyType,
+                       BOXAPIObjectKeyID,
+                       BOXAPIObjectKeySequenceID,
+                       BOXAPIObjectKeyETag,
+                       BOXAPIObjectKeySHA1,
+                       BOXAPIObjectKeyName,
+                       BOXAPIObjectKeyDescription,
+                       BOXAPIObjectKeySize,
+                       BOXAPIObjectKeyPathCollection,
+                       BOXAPIObjectKeyCreatedAt,
+                       BOXAPIObjectKeyModifiedAt,
+                       BOXAPIObjectKeyTrashedAt,
+                       BOXAPIObjectKeyPurgedAt,
+                       BOXAPIObjectKeyContentCreatedAt,
+                       BOXAPIObjectKeyContentModifiedAt,
+                       BOXAPIObjectKeyCreatedBy,
+                       BOXAPIObjectKeyModifiedBy,
+                       BOXAPIObjectKeyOwnedBy,
+                       BOXAPIObjectKeySharedLink,
+                       BOXAPIObjectKeyParent,
+                       BOXAPIObjectKeyItemStatus,
+                       //BOXAPIObjectKeyVersionNumber, excluded could reduce ~70% API response time
```

Comment:
Alternatively, could get fullFieldsArray and remove BOXAPIObjectKeyVersionNumber before returning it.

Advantage: One place where all fields are listed - no duplication, new fields automatically get into 'performant' listing.
Disadvantage: If a new field is introduced that is also non-performant, we need to know about this and remember to add a removal step for it as well.

What do you think?